dm/worker/subtask.go

```diff
@@ -245,18 +249,17 @@ retry:
 			if nu == nil {
 				// Now, when finished, it only stops the process
 				// if needed, we can refine to Close it
-				log.Infof("[subtask] %s all process units finished", st.cfg.Name)
+				st.l.Info("all process units finished")
 			} else {
-				log.Infof("[subtask] %s switching to next dm-unit %s", st.cfg.Name, nu.Type())
+				st.l.Info("switching to next unit", zap.Stringer("unit", cu.Type()))
 				st.setCurrUnit(nu)
 				// NOTE: maybe need a Lock mechanism for sharding scenario
 				st.run() // re-run for next process unit
 			}
 		case pb.Stage_Stopped:
 		case pb.Stage_Paused:
 			for _, err := range result.Errors {
-				log.Errorf("[subtask] %s dm-unit %s process error with type %v:\n %v",
-					st.cfg.Name, cu.Type(), err.Type, err.Msg)
+				st.l.Error("unit process error", zap.Stringer("unit", cu.Type()), zap.Reflect("error", err))
```

Comment:
```suggestion
				st.l.Error("unit process error", zap.Stringer("unit", cu.Type()), zap.Error(err))
```
`zap.Error(err)` is more simple ?

Reply:
it‘s not real error